src/variables/body/table.tsx

```diff
@@ -103,12 +113,12 @@ const useTbody = (items: Array<any>, defaultState: any) => {
             >
               <td style={{ paddingLeft: `${12}px`, width: `${25}%` }}>
                 <span
-                  className={`jp-Icon jp-Icon-16 ${setClassIcon(item.value)}`}
+                  className={`jp-Icon jp-Icon-16 ${setClassIcon(item.type)}`}
                 ></span>
                 {item.name}
               </td>
               <td style={{ paddingLeft: `${12}px`, width: `${75}%` }}>
-                {item.value}
+                {item.type}
```

Comment:
Shouldn't we keep `item.value` here?

Reply:
I check the response and type there is much better. The value probably we should insert in description

Reply:
But `item.type` returns something like `int`, that is shown in the "Value" column.

![image](https://user-images.githubusercontent.com/591645/67209574-e8e11100-f417-11e9-94d5-a5f2d8453f59.png)